rust/connlib/libs/client/src/file_logger.rs

```diff
@@ -29,7 +29,11 @@ pub fn layer<T>(
 where
     T: Subscriber + for<'a> tracing_subscriber::registry::LookupSpan<'a>,
 {
-    tracing::info!("Saving log files to: {}", log_dir.display());
+    #[cfg(debug_assertions)]
+    let level = LevelFilter::DEBUG;
+
+    #[cfg(not(debug_assertions))]
+    let level = LevelFilter::WARN;
```

Comment:
To be honest, I think this should simply be controlled by the application starting connlib. You can reuse `EnvFilter` and let the application pass in a string of what they want to log. See https://docs.rs/tracing-subscriber/0.3.17/tracing_subscriber/filter/struct.EnvFilter.html#method.new for parsing that even if it is not coming from an env variable.

I'd suggest that we have some meaningful defaults and apply the application-provided filter _afterwards_. That way, the app can override certain settings.

Reply:
I decided to open a new issue to track this, since it probably requires a bit more thought: https://github.com/firezone/firezone/issues/2067